### ISSUE TITLE
Order.LineItems.properties missing from LineItem

### ISSUE THREAD

**Issue by vincejairam** [nozzlegear/ShopifySharp#1088]
I don't see the properties [JsonProperty("properties")] as part of the LineItem class. It seems to be missing. I added the following

        [JsonProperty("properties")]
        public IEnumerable<NoteAttribute> LineProperties
        {
            get;
            set;
        }

to get those properties for additional instructions that the customer is requesting for the item in the store.

**Comment by nozzlegear** [comment]
Good catch! I just got back from vacation and plan to do a bunch of work on ShopifySharp in the next couple of days, I'll make sure this gets in there.
